app/pages/nonsipp/otherassetsdisposal/AnyPartAssetStillHeldPage.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.otherassetsdisposal
+
+import utils.RefinedUtils._
+import models.SchemeId.Srn
+import play.api.libs.json.JsPath
+import config.Refined.{Max50, Max5000}
+import pages.QuestionPage
+
+case class AnyPartAssetStillHeldPage(srn: Srn, assetIndex: Max5000, disposalIndex: Max50)
+    extends QuestionPage[Boolean] {
+
+  override def path: JsPath =
+    Paths.assetsDisposed \ toString \ assetIndex.arrayIndex.toString \ disposalIndex.arrayIndex.toString
+
+  override def toString: String = "portionStillHeld"
```

Comment:
For awareness @narendrapaduchuri @cyeniceri, the schema name for this field is `assets.otherAssets.otherAssetTransactions[n].assetsDisposed[n].fullyDisposedOf` (and it means the opposite than the UI collects here). It is still fine IMO to have it as @narendrapaduchuri created `AnyPartAssetStillHeldPage, portionStillHeld`, but when we do the BE transformation, the value should be negated.

Reply:
This is wrong now, @narendrapaduchuri you renamed `portionStillHeld` to `fullyDisposedOf`. Page name is still `AnyPartAssetStillHeldPage`. Let's have a chat.

Reply:
As discussed with @narendrapaduchuri , he is going to revert naming `fullyDisposedOf` in the next PR.